src/profile-logic/profile-data.js

```diff
@@ -1286,8 +1287,12 @@ export function getTimeRangeIncludingAllThreads(
   return completeRange;
 }
 
-export function defaultThreadOrder(threads: RawThread[]): ThreadIndex[] {
-  const threadOrder = threads.map((thread, i) => i);
+export function defaultThreadOrder(
```

Comment:
this function is only used in tracks.js, I wonder if it should be moved there and remove the `export`... indeed it's super specialized about that.